template.yaml

```diff
@@ -469,7 +469,10 @@ objects:
     dnsNames:
     - ${INTERCONNECT_APPLICATION_NAME}.${NAMESPACE}.svc
     - ${INTERCONNECT_APPLICATION_NAME}-${NAMESPACE}.${DEFAULT_ROUTE_DOMAIN}
-    - ${INTERCONNECT_APPLICATION_NAME}-console-${NAMESPACE}.${DEFAULT_ROUTE_DOMAIN}          
+    - ${INTERCONNECT_APPLICATION_NAME}-console-${NAMESPACE}.${DEFAULT_ROUTE_DOMAIN}
+    - ${INTERCONNECT_APPLICATION_NAME}-${REMOTE_ROUTER_HOST}.svc #FIXME mcostell bad hack to add back in a "second ca"
```

Comment:
I don't understand what you are trying to do here. These are the FQDNs represented by this service. it cannot both represent a local and a remote endpoint.

Reply:
So the idea behind the hack was that since the client ca points to this exact same secret that is based off of this certificate, simply signing the cert with both sets of dns names might be enough to get everyone to trust each other. Its a bad hack and only meant to deal with the complexity around not having a common ca: 
` - name: client-ca
          secret:
            secretName: ${INTERCONNECT_APPLICATION_NAME}-certs
        - name: sasl-config`